### PR TITLE
Fix more nits - prob. the last one of these PRs for a bit

### PR DESCRIPTION
**Pull request by Martin2112**
Commits should be independent and can be reviewed separately.

### Checklist

- [ ] I have updated the [CHANGELOG](CHANGELOG.md).
  - Adjust the draft version number according to [semantic versioning](https://semver.org/) rules.
- [ ] I have updated [documentation](docs/) accordingly.

### CODE REVIEW

File: storage/memory/log_storage.go [hunk 1]
```diff
@@ -427,11 +427,10 @@ func (t *readOnlyLogTX) GetUnsequencedCounts(ctx context.Context) (storage.Count
 	ret := make(map[int64]int64)
 	for id, tree := range t.ms.trees {
 		tree.RLock()
-		defer tree.RUnlock() // OK to hold until method returns.
-
 		k := unseqKey(id)
 		queue := tree.store.Get(k).(*kv).v.(*list.List)
 		ret[id] = int64(queue.Len())
+		tree.RUnlock() // OK to hold until method returns.
```
Comment (AlCutter) [inline storage/memory/log_storage.go]: Probably should remove the comment

Comment (Martin2112) [reply storage/memory/log_storage.go]: Good point, I'll fix that. The decrease in code coverage showed there are no tests for `RunInAdminSnapshot` so I'm going to fix that as well.

File: docs/merkletree/treetex/main.go [hunk 2]
```diff
@@ -382,5 +382,5 @@ func main() {
 	// arbitrary stream.
 	fmt.Print(preamble)
 	renderTree("", *treeSize, 0)
-	fmt.Println(postfix)
+	fmt.Print(postfix)
```
Comment (daviddrysdale) [inline docs/merkletree/treetex/main.go]: What's the nit here?  Won't this change the output?

File: storage/memory/log_storage.go (hunk 1 above)
Comment (daviddrysdale) [reply storage/memory/log_storage.go]: Still got the out-of-date comment.

File: docs/merkletree/treetex/main.go (hunk 2 above)
Comment (Martin2112) [reply docs/merkletree/treetex/main.go]: The extra newline is already in the string literal.

File: storage/memory/log_storage.go (hunk 1 above)
Comment (Martin2112) [reply storage/memory/log_storage.go]: Done.

**Pull request merged**